src/editor/components/modals/title-bar.jsx

```diff
@@ -11,14 +11,12 @@ const TitleBar = styled(AppBar)`
   background: ${THEME.clientModal.background};
 `;
 
-const Title = styled(Typography)`
-  color: #fff;
```

Comment:
I think doing `color: #fff !important;` should have the same effect and is a little cleaner than putting the styling inline in the component declaration.

Reply:
Neat suggestion!